clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/securitygroup/SecurityGroupLookupFactory.groovy

```diff
@@ -139,7 +139,11 @@ class SecurityGroupLookupFactory {
       }
       def describeSecurityGroupsRequest = new DescribeSecurityGroupsRequest().withGroupIds(groupId)
 
-      def securityGroups = amazonEC2.describeSecurityGroups(describeSecurityGroupsRequest).securityGroups
+      def securityGroups = []
+      try {
+        securityGroups = amazonEC2.describeSecurityGroups(describeSecurityGroupsRequest).securityGroups
+      } catch (Exception ignored) {}
+
```

Comment:
Turns out calling via `withGroupIds` will throw an exception if the `groupIds` don't actually exist.